### PR TITLE
Fix payments import

### PR DESCRIPTION
**Pull request by yrliou**
Fix https://github.com/brave/brave-browser/issues/3390

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Test Plan:
See the STP in https://github.com/brave/brave-browser/issues/3390

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

**Comment by yrliou** [comment]
run auto tests on mac, had some failures but unrelated to this PR:
test-security failed because of jest;
autoplay related tests crashed:
AutoplayPermissionContextTests.TestInsecureQueryingUrl, AutoplayPermissionContextTests.TestNonAutoRefresh, BraveContentSettingsObserverAutoplayTest.AllowAutoplay

File: browser/importer/brave_profile_writer.cc [hunk 1]
```diff
@@ -106,13 +106,13 @@ void BraveProfileWriter::SetBridge(BraveInProcessImporterBridge* bridge) {
   bridge_ptr_ = bridge;
 }
 
-void BraveProfileWriter::OnWalletInitialized(brave_rewards::RewardsService*
-  rewards_service, int error_code) {
-  if (error_code) {
+void BraveProfileWriter::OnWalletInitialized(
+    brave_rewards::RewardsService* rewards_service, int result) {
+  if (result != 0 && result != 12) {  // 12: ledger::Result::WALLET_CREATED
```
Comment (bsclifton) [inline browser/importer/brave_profile_writer.cc]: Would be better to use the constants... but I know I ran into issues trying to do that (including the headers breaks the build)

Comment (yrliou) [reply browser/importer/brave_profile_writer.cc]: Although I would prefer enum here, I'm not sure we could use ledger::Result which was defined in bat-native ledger lib in other components. AFAIK, we're also using int values directly in brave_rewards_ui.cc.
I'm going to merge this PR first to fix the bug, we could probably refactor this part along with similar stuff in brave_rewards_ui.cc later.

Comment (yrliou) [reply browser/importer/brave_profile_writer.cc]: cc @NejcZdovc ^^

Comment (NejcZdovc) [reply browser/importer/brave_profile_writer.cc]: @bridiver can tell you more about it, but we only use ledger lib inside rewards folder. For everything else we create duplicate definition inside core

**Pull request merged**